src/runtime/rpc/rpc_module.cc

```diff
@@ -395,6 +395,16 @@ PackedFunc WrapTimeEvaluator(PackedFunc pf, Device dev, int number, int repeat,
         }
         t->Stop();
         int64_t t_nanos = t->SyncAndGetElapsedNanos();
+
+        if ((min_repeat_ms > 0) && (t_nanos == 0)) {
+          // We're almost certainly caught in an infinite loop, perahsp because something
+          // in the timing mechanism isn't working as expected.
+          LOG(FATAL) << __FILE__ << ":" << __LINE__ << ": "
```

Comment:
`LOG(FATAL)` already contains file and line information, so you don't need to add it here.

Reply:
Reasonable assumption, but actually it wasn't.  I've fixed this by enabling customized logging when this file is compiled for Hexagon.